packages/protocol/scripts/ganache.ts

```diff
@@ -0,0 +1,98 @@
+/* tslint:disable: no-console */
+import * as ganache from '@celo/ganache-cli'
```

Comment:
Wondering if we need this script given that https://github.com/celo-org/celo-monorepo/pull/462 fixed the ganache issue.

Reply:
reverted